src/ctapipe/utils/astro.py

```diff
@@ -4,63 +4,194 @@
 not provided by external packages and/or to satisfy particular needs of
 usage within ctapipe.
 """
-from astropy import units as u
+
+import logging
+from copy import deepcopy
+from enum import Enum
+
+import astropy.units as u
 from astropy.coordinates import Angle, SkyCoord
+from astropy.table import Table
+from astropy.time import Time
 
-__all__ = ["get_bright_stars"]
+log = logging.getLogger("main")
 
+__all__ = ["get_star_catalog", "get_bright_stars"]
 
-def get_bright_stars(pointing=None, radius=None, magnitude_cut=None):
-    """
-    Get an astropy table of bright stars.
 
-    This function is using the Yale bright star catalog, available through ctapipe
-    data downloads.
+class StarCatalogues(Enum):
```

Comment:
I find it strange to use mutable values as value of an enum.

I'd define the enum like this:

```
class StarCatalog(Enum):
    Yale = auto()
    Hipparcos = auto()
```

and then define the settings as:

```
CATALOG_PROPERTIES = {
    StarCatalog.Hiparcos: {...},
    StarCatalog.Yale: {...},
}
```

I naively tried `get_star_catalog(StarCatalogues.Hipparcos)` which resulted in a very, very unweildy error:

```
KeyError: <StarCatalogues.Hipparcos: {'directory': 'I/239/hip_main', 'coordinates': {'frame': 'icrs', 'epoch': 'J1991.25', 'RA': {'column': 'RAICRS', 'unit': 'deg'}, 'DE': {'column': 'DEICRS', 'unit': 'deg'}, 'pmRA': {'column': 'pmRA', 'unit': 'mas/yr'}, 'pmDE': {'column': 'pmDE', 'unit': 'mas/yr'}}, 'columns': ['RAICRS', 'DEICRS', 'pmRA', 'pmDE', 'Vmag', 'BTmag', 'HIP'], 'record': 'hipparcos_star_catalog'}>
```
```